yarn-project/bb-prover/src/bb/execute.ts

```diff
@@ -393,8 +395,9 @@ export async function verifyAvmProof(
   proofFullPath: string,
   verificationKeyPath: string,
   log: LogFn,
+  calldataPath?: string,
```

Comment:
This probably needs to be always passed (i.e., without "?")

Reply:
With passing calldata as part of the proof, this change was reverted.